omero/sysadmins/client-server-ssl.rst

```diff
@@ -8,34 +8,34 @@ OMERO.server and clients do not automatically support host verification, so a
 is possible.
 This may result in users inadvertently transmitting their login credentials to an attacker.
 
-This can be remedied by configuring OMERO.server with a certificate (the same certificate used for OMERO.web Nginx may work), and ensuring all OMERO clients are configured to verify the server certificate before connecting.
+This can be remedied by configuring OMERO.server with a certificate and ensuring all OMERO clients are configured to verify the server certificate before connecting.
 
 
 Server certificate
 ------------------
 
-The easiest solution is to re-use the SSL certificates used to protect OMERO.web.
-First convert the public certificate :file:`server.pem` and private key :file:`server.key` to the PKCS12 format where ``secret`` is the password used to protect the combined output file :file:`server.p12`::
+The easiest solution is to use the `omero-certificates <https://github.com/ome/omero-certificates>`_ plugin to
+generate self-signed server certificates alongside their associated configuration.
+
+It is also possible to re-use the SSL certificates used to protect OMERO.web. First convert
```

Comment:
```suggestion
It is also possible to re-use the SSL certificates used to protect OMERO.web. First convert
```

Really unsure about 2 things:
Is the following paragraph describing how to ``re-use teh SSL certificates used to protect OMERO.web`` or, rather, as described above, is describing how to use the omero-certificates plugin ? If the former, where is a description about how to use the plugin ? Is the configuration example (using the Glacier2) described below valid for both ways ?

Reply:
> Is the following paragraph describing how to re-use teh SSL certificates used to protect OMERO.web or, rather, as described above, is describing how to use the omero-certificates plugin ? 

To the best of my knowledge the former. Maybe we need to dwelve back into history here to understand the assumptions and make decisions:
- this page was originally introduced in https://github.com/ome/omero-documentation/pull/1942 in June 2019 on the OMERO 5.5.0 line
- the initial work on omero-certificates and the first release was a year later in March 2020 - see https://github.com/ome/omero-certificates/releases/tag/0.1.0

So the content of this page has been largely superseded by the `omero-certificates` plugin which  runs several of the commands from the `Server certificate` section under the hood. The only notable difference is that the `Server certificate` section explains how to re-use an existing public certificate/private key e.g. used by Nginx rather than generating a self-signed certificate like the plugin dows

>  If the former, where is a description about how to use the plugin ?

That's probably the key issue: there is no canonical place to the best of my knowledge. At the moment:
- each of the server installation pages include a section explaining how to run the certificates - see https://omero.readthedocs.io/en/stable/sysadmins/unix/server-centos7-ice36.html#configuring-omero-server
- this PR updates the section of the upgrade page to update the certificate settings

>  Is the configuration example (using the Glacier2) described below valid for both ways ?

Largely yes. This PR adjusts the example to make it compatible with the settings of the latest `omero certificates`

Reply:
That makes sense, thank you.